src/server/scripts/Commands/cs_ban.cpp

```diff
@@ -389,14 +389,14 @@ class ban_commandscript : public CommandScript
 
         if (filter.empty())
         {
-            LoginDatabasePreparedStatement* stmt = LoginDatabase.GetPreparedStatement(LOGIN_SEL_ACCOUNT_BANNED_ALL);
-            result = LoginDatabase.Query(stmt);
+            LoginDatabasePreparedStatement* stmt2 = LoginDatabase.GetPreparedStatement(LOGIN_SEL_ACCOUNT_BANNED_ALL);
+            result = LoginDatabase.Query(stmt2);
         }
         else
         {
-            LoginDatabasePreparedStatement* stmt = LoginDatabase.GetPreparedStatement(LOGIN_SEL_ACCOUNT_BANNED_BY_USERNAME);
-            stmt->setString(0, filter);
-            result = LoginDatabase.Query(stmt);
+            LoginDatabasePreparedStatement* stmt2 = LoginDatabase.GetPreparedStatement(LOGIN_SEL_ACCOUNT_BANNED_BY_USERNAME);
+            stmt2->setString(0, filter);
+            result = LoginDatabase.Query(stmt2);
         }
```

Comment:
` result = LoginDatabase.Query(stmt2);` you can write this only once and outside of the `if / else`. I think that the stmt2 variable remains outside the **if/else**

Reply:
no, because `stmt2` would be out of scope